phnt/include/ntldr.h

```diff
@@ -1378,12 +1594,12 @@ NTSYSAPI
 NTSTATUS
 NTAPI
 LdrResolveDelayLoadsFromDll(
-    _In_ PVOID ParentModuleBase,
-    _In_ PCSTR TargetDllName,
+    _In_ PVOID ImportModuleBase,
+    _In_ PCSTR ExportDllName,
```

Comment:
ImportModuleBase needs to include DllBase or BaseAddress instead of ModuleBase

ExportDllName should probably remain TargetDllName? The first variable named Import, and the second variable named as Export is implying you pass the address of an image with imports as the first and the address of an image with exports as second?